test/HexFloat.cpp

```diff
@@ -1054,8 +1054,8 @@ FloatParseCase<T> GoodFloatParseCase(std::string literal, bool negate_value,
 INSTANTIATE_TEST_CASE_P(FloatParse, ParseNormalFloatTest,
                         ::testing::ValuesIn(std::vector<FloatParseCase<float>>{
                             // Failing cases due to trivially incorrect syntax.
-                            BadFloatParseCase<float>("abc", false),
-                            BadFloatParseCase<float>("abc", true),
+                            BadFloatParseCase<float>("abc", false, 0.0f),
```

Comment:
True for the float case, and I'll fix that.  Not so for the Float16 since I'm supplying uint16_t constructor arguments instead of Float16.  Doesn't seem much of a win to change

```BadFloatParseCase<Float16>("abc", false, uint16_t{0})```
to
```BadFloatParseCase("abc", false, Float16(uint16_t{0}))```